ore-rs.gemspec

```diff
@@ -16,14 +16,22 @@ Gem::Specification.new do |s|
 
   s.authors  = ["Matt Palmer"]
   s.email    = ["matt@cipherstash.com"]
-  s.homepage = "https://github.com/cipherstash-ruby-ore-rs"
+  s.homepage = "https://cipherstash.com"
```

Comment:
Don't we want the homepage to be the git repo?

Reply:
From convo offline: we want this to be the ActiveStash landing page once it's ready (across all of our gems). I'm making this the CipherStash homepage until the landing page is ready.

Reply:
I have very strong opinions that the homepage for gems only tangentially-related to ActiveStash should *not* be ActiveStash.  If that is the decision, though, I think it'd be best just to set that to be the homepage now, and draft this PR until that page is ready to go.  There's very little point in making a release of `ore-rs` today to change the homepage URL, and then making another one in a day or two's time to change the homepage URL again.